nailgun/entities.py

```diff
@@ -4283,6 +4283,59 @@ def create(self, create_missing=None):
         ).read()
 
 
+class RecurringLogic(
+        Entity,
+        EntityReadMixin):
+    """A representation of a Recurring logic entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'cron_line': entity_fields.StringField(),
+            'end_time': entity_fields.DateTimeField(),
+            'iteration': entity_fields.IntegerField(),
+            'task_group_id': entity_fields.IntegerField(),
+            'state': entity_fields.StringField(),
```

Comment:
Missing the `tasks` field which is present in read_json output. OneToManyField(ForemanTask), most likely.